deps/wazuh_testing/wazuh_testing/end_to_end/monitoring.py

```diff
@@ -129,7 +137,7 @@ def filter_events_by_timestamp(match_events: List) -> List:
 
                 sleep(scan_interval)
 
-                current_timeout += scan_interval
+                current_timeout = current_timeout + scan_interval
```

Comment:
Why has this addition been separated?

Reply:
Changes from https://github.com/wazuh/wazuh-qa/pull/4959. Restored in newer changes